Microsoft.NET.Build.Containers/ParseContainerProperties.cs

```diff
@@ -57,16 +58,51 @@ public ParseContainerProperties()
         ParsedContainerTag = "";
         NewContainerRegistry = "";
         NewContainerImageName = "";
-        NewContainerTag = "";
+        NewContainerTags = Array.Empty<string>();
+    }
+
+    private static bool ParsePropertyAsArray(string input, [NotNullWhen(true)] out string[]? items) {
```

Comment:
I _think_ there might be a builtin for this? In any case it's safe to just split on `;` here because `;` isn't valid for tag strings.

Reply:
perfect, I thought there must be something 👍